src/NzbDrone.Core/Datastore/Migration/Framework/NzbDroneSQLiteProcessor.cs

```diff
@@ -1,28 +1,30 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Data;
 using System.Linq;
-using FluentMigrator;
 using FluentMigrator.Expressions;
 using FluentMigrator.Model;
-using FluentMigrator.Runner;
-using FluentMigrator.Runner.Announcers;
 using FluentMigrator.Runner.Generators.SQLite;
 using FluentMigrator.Runner.Processors.SQLite;
 using System.Text.RegularExpressions;
+using Microsoft.Extensions.Logging;
```

Comment:
nitpick: reorder usings. (was already unordered, hence vs adding them to the end)